Bio/PDB/PDBIO.py

```diff
@@ -294,7 +294,7 @@ def _get_atom_line(
             return _PQR_ATOM_FORMAT_STRING % args
 
     # Public methods
-    def save(self, file, select=_select, write_end=True, preserve_atom_numbering=False):
+    def save(self, file, select=_select, write_end=True, preserve_atom_numbering=False, write_ter = True):
```

Comment:
This should be `write_ter=True`, per style guidelines, but the style check should catch this.